src/context/Theme.tsx

```diff
@@ -146,39 +190,40 @@ const lightMode: PaletteOptions = {
     },
     contrastThreshold,
     error: {
-        alpha_05: alpha(errorMain, 0.05),
-        alpha_12: alpha(errorMain, 0.12),
-        alpha_26: alpha(errorMain, 0.26),
-        alpha_50: alpha(errorMain, 0.5),
+        alpha_05: `rgba(${rgbTranslations.light.error.main}, 0.05)`,
```

Comment:
Noting down what was said during dailies.

We'll want to eventually make something that helps generate the `rgba` string.